sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1486,16 +1534,11 @@ public void setProfilingEnabled(boolean profilingEnabled) {
    * @return the profiling traces dir. path or null if not set
    */
   public @Nullable String getProfilingTracesDirPath() {
-    return profilingTracesDirPath;
-  }
-
-  /**
-   * Sets the profiling traces dir. path
-   *
-   * @param profilingTracesDirPath the profiling traces dir. path
-   */
-  public void setProfilingTracesDirPath(@Nullable String profilingTracesDirPath) {
-    this.profilingTracesDirPath = profilingTracesDirPath;
+    final String cacheDirPath = getCacheDirPath();
```

Comment:
`setProfilingTracesDirPath` is removed but `getProfilingTracesDirPath` still returns the same value, similar to `getOutboxPath` since its always a subdirectory.